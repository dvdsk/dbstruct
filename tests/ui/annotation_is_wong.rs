use dbstruct::dbstruct;

#[dbstruct]
struct Test {
    #[dbstruct(NotAnnotation)]
    field: u8,
}

fn main() {}
