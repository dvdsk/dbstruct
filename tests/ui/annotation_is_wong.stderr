error: Not a dbstruct wrapper annotation (try DefaultTrait or DefaultValue)
 --> tests/ui/annotation_is_wong.rs:5:16
  |
5 |     #[dbstruct(NotAnnotation)]
  |                ^^^^^^^^^^^^^

error: custom attribute panicked
 --> tests/ui/annotation_is_wong.rs:3:1
  |
3 | #[dbstruct]
  | ^^^^^^^^^^^
  |
  = help: message: not yet implemented
